test/integration/parachain/staging/setup/initialize.test.ts

```diff
@@ -187,4 +279,138 @@ describe("Initialize parachain state", () => {
         }
         assert.isTrue(isNominationEnabled);
     });
+
+    it("should set collateral ceiling and thresholds for aUSD", async function () {
+        // only get aUSD if it hasn't been set yet (e.g. when running this test in isolation)
+        !aUSD ? (aUSD = await getAUSDForeignAsset(sudoInterBtcAPI.assetRegistry)) : undefined;
+
+        if (aUSD === undefined) {
+            // no point in completing this if aUSD is not registered
+            this.skip();
+        }
+
+        const sudoThis = async (extrinsic: SubmittableExtrinsic<"promise", ISubmittableResult>) =>
+            sudoInterBtcAPI.api.tx.sudo.sudo(extrinsic).signAndSend(sudoAccount);
+
+        // (unsafely) get first collateral currency's ceiling and thresholds
+        const existingCollCcy = getCorrespondingCollateralCurrencies(sudoInterBtcAPI.getGovernanceCurrency())[0];
+        const existingCcyPair = newVaultCurrencyPair(api, existingCollCcy, sudoInterBtcAPI.getWrappedCurrency());
+        // borrow values from existing currency pair
+        const [optionCeilValue, existingSecureThreshold, existingPremiumThreshold, existingLiquidationThreshold] =
+            await Promise.all([
+                sudoInterBtcAPI.api.query.vaultRegistry.systemCollateralCeiling(existingCcyPair),
+                sudoInterBtcAPI.vaults.getSecureCollateralThreshold(existingCollCcy),
+                sudoInterBtcAPI.vaults.getPremiumRedeemThreshold(existingCollCcy),
+                sudoInterBtcAPI.vaults.getLiquidationCollateralThreshold(existingCollCcy),
+            ]);
```

Comment:
Why not just set some arbitrary values?

Reply:
> Why not just set some arbitrary values?

No good reason. I just wanted some values that have a "realistic spread" between the thresholds.